### PR TITLE
Load CSS common to repositories in same directory

### PR DESCRIPTION
**Pull request by nboisteault**
* Ticket : #1756

### CODE REVIEW

File: lizmap/modules/view/controllers/lizMap.classic.php [hunk 1]
```diff
@@ -312,18 +312,39 @@ function f($x)
...
+        // Also if media/themes/default/css is found one directory above repositorie's one
+        // it will apply to all repositories
         if ($lrep->getData('allowUserDefinedThemes')) {
             $repositoryPath = $lrep->getPath();
             $cssArray = array('main', 'map', 'media');
             $themeArray = array('default', $project);
             foreach ($cssArray as $k) {
+                // Handle theme applying to all repositorie's projects in the same directory
+                $cssRelPath = '../media/themes/default/css/' . $k . '.css';
+                $cssPath = $lrep->getPath() . '/' . $cssRelPath;
```
Comment (rldhont) [inline lizmap/modules/view/controllers/lizMap.classic.php:331]:
```suggestion
                $cssPath = realpath($repositoryPath.$cssRelPath);
```

File: lizmap/modules/view/controllers/lizMap.classic.php [hunk 2]
```diff
@@ -312,18 +312,39 @@ function f($x)
...
+                    //~ $rep->addCssLink( $cssUrl );
+                    // Use addHeadContent and not addCssLink to be sure it will be loaded after minified code
+                    $rep->addHeadContent('<link type="text/css" href="' . $cssUrl . '" rel="stylesheet" />');
+                }
+
+                // Handle themes applying to a repository (media/themes/default)
+                // or to a project (media/themes/PROJECT-NAME)
                 foreach ($themeArray as $theme) {
                     $cssRelPath = 'media/themes/'.$theme.'/css/'.$k.'.css';
                     $cssPath = $lrep->getPath().'/'.$cssRelPath;
```
Comment (rldhont) [inline lizmap/modules/view/controllers/lizMap.classic.php:350]:
```suggestion
                $cssPath = realpath($repositoryPath.$cssRelPath);
```

**Review by rldhont — APPROVED** [review]
Some little improve suggestion

**Comment by nboisteault** [comment]
@laurentj gentle ping

**Comment by nboisteault** [comment]
@laurentj waiting for your review to merge. Thanks.

**Review by laurentj — APPROVED** [review]

**Comment by rldhont** [comment]
Thanks @laurentj

**Pull request merged**